src/pages/ExamsPage/components/AllowanceListActions.jsx

```diff
@@ -0,0 +1,88 @@
+import {
+  ActionRow,
+  Icon,
+  IconButtonWithTooltip,
+  ModalDialog,
+  StatefulButton,
+  useToggle,
+} from '@openedx/paragon';
+import { DeleteOutline, EditOutline } from '@openedx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import messages from '../messages';
+import { useDeleteAllowance } from '../hooks';
+
+const DeleteModal = (isOpen, onCancel, onDelete, formatMessage) => (
+  <ModalDialog
+    isOpen={isOpen}
+    onClose={onCancel}
+    variant="default"
+    hasCloseButton
+    isFullscreenOnMobile
+  >
+    <ModalDialog.Header>
+      <ModalDialog.Title>
+        {formatMessage(messages.deleteAllowanceHeader)}
+      </ModalDialog.Title>
+    </ModalDialog.Header>
+    <ModalDialog.Body>
+      <p>{formatMessage(messages.deleteAllowanceBody)}</p>
+    </ModalDialog.Body>
+    <ModalDialog.Footer>
+      <ActionRow>
+        <ModalDialog.CloseButton variant="tertiary" onClick={onCancel}>
+          {formatMessage(messages.deleteAllowanceCancel)}
+        </ModalDialog.CloseButton>
+        <StatefulButton
+          variant="primary"
+          state=""
+          labels={{
+            default: formatMessage(messages.deleteAllowanceDelete),
+          }}
+          onClick={onDelete}
+        />
+      </ActionRow>
+    </ModalDialog.Footer>
+  </ModalDialog>
+);
+
+const AllowanceListActions = (allowance) => {
+  const { formatMessage } = useIntl();
+  const [isDeleteModalOpen, setDeleteModalOpen, setDeleteModalClosed] = useToggle(false);
+
+  const deleteAllowance = useDeleteAllowance();
+
+  const handleDelete = () => {
+    deleteAllowance(allowance.id);
+    setDeleteModalClosed();
+  };
+
+  return (
+    <div className="allowances-actions text-right">
+      <IconButtonWithTooltip
+        tooltipPlacement="top"
+        tooltipContent={formatMessage(messages.editAllowanceButton)}
+        src={EditOutline}
+        iconAs={Icon}
+        alt={formatMessage(messages.editAllowanceButton)}
+        onClick={setDeleteModalOpen}
```

Comment:
This will eventually be replaced with the edit allowance modal I'm working on correct?

Reply:
yeah I just put this in there so it was hooked up to something